packages/terra-time-input/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "terra-time-input",
+  "main": "lib/TimeInput.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "The terra-time-input component is an input for entering time. During time entry it uses [text-mask](https://github.com/text-mask/text-mask) to mask the input value to conform to the time format.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-time-input",
+    "TimeInput",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.x"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^0.4.1",
+    "terra-mixins": "^1.1.1"
+  },
+  "dependencies": {
+    "moment": "^2.17.1",
+    "prop-types": "^15.5.8",
+    "react-text-mask": "^5.0.2",
```

Comment:
https://github.com/text-mask/text-mask/issues/510#issuecomment-301098488
^ This describes the warning we were seeing before and how it was fixed.